### PR TITLE
OSOE-525: Collapse spelling annotation

### PR DESCRIPTION
**Pull request by davidpuplava**
[OSOE-525](https://lombiq.atlassian.net/browse/OSOE-525)

[OSOE-525]: https://lombiq.atlassian.net/browse/OSOE-525?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by davidpuplava** [comment]
FYI, this PR is not ready to merge even though "All the checks have passed" - the reason is that the spelling-this-repo.yml workflow file is invalid (I accidentally broke it in this branch). I wonder if there is a way to mark the PR as failed when there are invalid workflows.

**Comment by wAsnk** [comment]
It's okay, nobody will merge it or review it until it is not requested. Don't worry about that!

**Comment by davidpuplava** [comment]
Excellent - successfully commented and then subsequently collapsed the spelling annotations on this PR

File: .github/workflows/spelling.yml [hunk 1]
```diff
@@ -73,15 +75,34 @@ jobs:
...
+      pull-requests: write
+    if: always() && needs.check-spelling.outputs.followup && github.event_name == 'pull_request'
+    steps:
+      - name: Comment (PR)
+        uses: Lombiq/GitHub-Actions/.github/actions/spelling@issue/OSOE-525
+        with:
+          dictionary-source-prefixes: ${{ inputs.dictionary-source-prefixes }}
+          extra-dictionaries: ${{ inputs.extra-dictionaries }}
+          config: ${{ inputs.config }}
+          spell-check-this: ${{ inputs.spell-check-this }}
```
Comment (wAsnk) [inline .github/workflows/spelling.yml]: All these are needed in this job as well when posting/hiding comment?

**Review by wAsnk — CHANGES_REQUESTED** [review]

File: .github/workflows/spelling.yml (hunk 1 above)
Comment (davidpuplava) [reply .github/workflows/spelling.yml]: Good question - let me try testing without these extra inputs.

Comment (davidpuplava) [reply .github/workflows/spelling.yml]: Testing in a private repo, I confirmed that these extra inputs are not necessary. I've pushed a new commit.

**Review by wAsnk — APPROVED** [review]

**Pull request merged**